### DISCUSSION TITLE
de/1-06-ai-tools-services/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#50] [General]
# de/1-06-ai-tools-services/

https://ai.lernos.org/de/1-06-ai-tools-services/

**Comment by AntaresReisky** [comment]
https://www.aiva.ai für Musik Komposition

**Comment by Marisibisi** [comment]
Miro wird bei unserem Unternehmen - ansässig in Österreich - als nicht DSGVO konform eingestuft.

**Comment by tedmcrumble** [comment]
MeetGeek als Kalenderintegration für die Zusammenfassung von Online-Meetings könnte unter weiteren Anwendungen ergänzt werden

**Comment by KathaZ** [comment]
A2/Z6: *bietet Unternehmenskunden*
A2/Z9: Microsoft*-*Produkten
A4/Z7: von GPT-4; Funktionsumfang (Unterstriche löschen)
Abschnitt "KI-Tools für Bilder/Z3: (oder Videos)*,*
Abschnitt "KI-Tools für Video/Z4: KI-unterstütz*t*en Funktionen
